### PR TITLE
Update mondo.md

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
add new publication
address https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1826

### CODE REVIEW

File: ontology/mondo.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ usages:
       - url: https://monarchinitiative.org/phenotype/HP:0001300#disease
         description: "Parkinsonism: Characteristic neurologic anomaly resulting form degeneration of dopamine-generating cells in the substantia nigra, a region of the midbrain, characterized clinically by shaking, rigidity, slowness of movement and difficulty with walking and gait."
     publications:
+      - id: https://medrxiv.org/cgi/content/short/2022.04.13.22273750v1
```
Comment (cthoyt) [inline ontology/mondo.md:54]:
```suggestion
      - id: https://www.medrxiv.org/content/10.1101/2022.04.13.22273750
```

**Comment by cthoyt** [comment]
Blocked by #1855

File: ontology/mondo.md (hunk 1 above)
Comment (cthoyt) [reply ontology/mondo.md:54]: Btw it looks like this should actually be a top-level entry and not inside the usage

**Review by cthoyt — APPROVED** [review]

**Review by cthoyt — CHANGES_REQUESTED** [review]
Location of publication needs updating

**Review by matentzn — APPROVED** [review]

**Comment by nicolevasilevsky** [comment]
thank you both!

**Pull request merged**